elastic/kibana
# [ResponseOps][OnWeek] Connector logs view

Resolves https://github.com/elastic/kibana/issues/147795

## Summary

Adds a connectors event log tab that gives the ability to see historical activity of connectors.


### Checklist

Delete any items that are not applicable to this PR.


### To verify

- Create a rule with a connector, create a few to help test the search capability
- Go to http://localhost:5601/app/management/insightsAndAlerting/triggersActionsConnectors/logs
- Verify that you can see the execution logs for the connector
- Verify that you can search
- Verify that you can filter by error/success
